utils/luci-mod-freifunk-ui/luasrc/view/freifunk/index.htm

```diff
@@ -16,12 +16,13 @@
 local ipkg = require "luci.model.ipkg"
 
 -- only redirect if assistent is installed and no root password is set
--- we use isfile because the view is run as root and we can't use e.g.
--- ipkg.installed or checkpasswd
+-- we use isfile because the view is not run as root and we can't use
```

Comment:
i should have checked this before ... did not believe that we are so much apart from upstream